sideboard/config.py

```diff
@@ -38,36 +99,64 @@ def get_dirnames(pyname, plugin):
     package under the configured "plugins_dir" directory.
 
     These two directories are also automatically inserted into plugin config
-    files as "root" and "module_root" and are available for interpolation.  For
-    example, a plugin could have a line in their config file like
+    files as "root" and "module_root" and are available for interpolation. For
+    example, a plugin could have a line in their config file like::
+
         template_dir = "%(module_root)s/templates"
+
+
     and that would be interpolated to the correct absolute path.
 
-    This function takes two parameters:
-    pyname: the __file__ of the module needing to know these directories
-    plugin: boolean indicating whether this is a plugin or Sideboard itself
+    Parameters:
+        requesting_file_path (str): The __file__ of the module requesting the
+            "root" and "module_root" directories.
+        is_plugin (bool): Indicates whether a plugin is making the request or
+            Sideboard itself is making the request.
     """
-    module_dir = os.path.dirname(os.path.abspath(pyname))
-    if plugin:
+    module_dir = os.path.dirname(os.path.abspath(requesting_file_path))
+    if is_plugin:
         from sideboard.lib import config
         plugin_name = os.path.basename(module_dir)
-        root_dir = os.path.join(config['plugins_dir'], plugin_name.replace('_', '-'))
+        root_dir = os.path.join(config['plugins_dir'], plugin_name)
     else:
         root_dir = os.path.realpath(os.path.join(module_dir, '..'))
     return module_dir, root_dir
 
 
-def get_config_files(requesting_file_path, plugin):
+def get_config_files(requesting_file_path, is_plugin):
     """
-    Returns a list of absolute paths to config files to be parsed by ConfigObj,
-    which allows subsequent files to override values in earlier files.  We parse
-    the following files in this order:
-    -> development-defaults.ini, which can be checked into source control and
-        include whatever we want to be present in a development environment
-    -> development.ini, which shouldn't be checked into source control, allowing
-        a developer to include local settings not shared with others
-    -> /etc/sideboard/plugins.d/<PLUGIN_NAME>.cfg, which is the config file we
-        expect in production; the others shouldn't exist on a real install
+    Returns a list of absolute paths to config files for the given file path.
+
+    When the returned config files are parsed by ConfigObj each subsequent
+    file will override values in earlier files.
+
+    If `is_plugin` is `True` the first of the returned files is:
+
+    * /etc/sideboard/plugins.d/<PLUGIN_NAME>.cfg, which is the config file we
+      expect in production
+
+
+    If `is_plugin` is `False` the first of the returned files is:
```

Comment:
I should note, I changed the `plugin` parameter to `is_plugin` for clarity. Generally I'm fine with not using the `is_*` prefix to indicate a boolean, but for these methods in particular I was initially confused because I kept thinking that `plugin` was referring to a module, and I had keep going back to read the docstring.